manifests/0000_25_kube-controller-manager-operator_01_config.crd.yaml

```diff
@@ -30,216 +29,6 @@ spec:
             submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
           type: string
         metadata:
-          description: ObjectMeta is metadata that all persisted resources must have,
```

Comment:
why it was generating ObjectMeta before?

Reply:
I think it's caused by older generator, but @damemi will know better.